plugins/web/active/Old_Backup_and_Unreferenced_Files@OWTF-CM-006.py

```diff
@@ -4,15 +4,17 @@
 https://www.owasp.org/index.php/Testing_for_Old,_Backup_and_Unreferenced_Files_(OWASP-CM-006)
 """
 
-DESCRIPTION = "Active probing for juicy files (DirBuster)"
+DESCRIPTION = "Active probing for juicy files (SVN-Extractor + DirBuster)"
 
 def run(PluginInfo):
 	#ServiceLocator.get_component("config").Show()
 	# Define DirBuster Commands to use depending on Interaction Setting:
 	# DirBuster allows much more control when interactive
 	# DirBuster can also be run non-interactively for scripting
 	DirBusterInteraction = { 'true' : 'DirBusterInteractive', 'false' : 'DirBusterNotInteractive' }
-	Content = ServiceLocator.get_component("plugin_helper").CommandDump('Test Command', 'Output', ServiceLocator.get_component("resource").GetResources(DirBusterInteraction[ServiceLocator.get_component("db_config").Get('INTERACTIVE')]), PluginInfo, [])
+	Content = ServiceLocator.get_component("plugin_helper").CommandDump('Test Command', 'Output', ServiceLocator.get_component("resource").GetResources('SVN_Extractor'), PluginInfo, [])
+	Content += ServiceLocator.get_component("plugin_helper").CommandDump('Test Command', 'Output', ServiceLocator.get_component("resource").GetResources('SVN_Extract_URLs'), PluginInfo, [])
```

Comment:
@delta24 problem is as you can see there are two separate lines of extract_urls one for svn and other for dirbuster and since dirbuster is executed later so the extracted urls of svn are replaced by that of dirbuster. What to do ?